submissions/opeyemiPeter/DOM/signup.js

```diff
@@ -0,0 +1,98 @@
+let form = document.getElementById("signup_form"),
+  firstName = document.getElementById("first_name"),
+  lastName = document.getElementById("last_name"),
+  phoneNumber = document.getElementById("phone_number"),
+  formEmail = document.getElementById("form_email"),
+  password = document.getElementById("password"),
+  confirmPassword = document.getElementById("confirm_password");
+
+function validation(field, regExp, errorMess1, errorMess2, event) {
+  event.preventDefault();
+  if (field.value === "") {
+    field.nextElementSibling.innerHTML = errorMess1;
+    event.preventDefault();
+    return false;
+  }
+  if (!field.value.match(regExp)) {
+    field.nextElementSibling.innerHTML = errorMess2;
+    event.preventDefault();
+    return false;
+  } else {
+    field.nextElementSibling.innerHTML = "";
+    return true;
+  }
+}
+
+function passwordValidation(password1, password2, event) {
+  const passwordRegex = /(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+
+  if ((password1.value && password2.value) === "") {
```

Comment:
I tot I spoke about this in the chat we had. This condition can be handled in the function above

Reply:
yes, you did, but it didnt work out. that why i resorted into this. due to the number parameters the password function expects to passed in

Reply:
@adaugochi , i will appreciate if i can see how to implement it, the password validation function takes in 3 parameters while the validation function takes in 5 parameters so i dont know how to rightly implement, but i dont understand the concept just the implementation.

i have made the  'else if 'correction.
thanks

Reply:
It also takes 5. `validation(password1, passwordRegex, 'Please enter your password', 'Your password should have a minimum of 6 characters, 1 capital letter, 1 special character eg @ and 1 number', event)`. How is it not taking 5?

Reply:
@adaugochi thanks i changed it now, you can now view.